tests/tx_3_2.at

```diff
@@ -21,6 +21,99 @@ xa_close/0
 AT_CHECK([export LIXA_PROFILE=CASE_PROF_0000 ; lixa_test_exec.sh reset recycle case0000 2>&1 | tee $TESTS_TMP_FILE1 ; grep 'lixa_monkeyrm_close' $TESTS_TMP_FILE1 | grep 'ret_cod' | wc -l | grep 3], [0], [ignore], [ignore])
 AT_CLEANUP
 
+AT_SETUP([TX/3.2/0.1])
+# As for TX/3.2/0.0 but profile CASE_PROF_0002 is specified by the test program
+AT_DATA([monkey1s.conf],
+[[# first monkey R.M. config
+xa_open/0
+xa_close/0
+]])
+AT_DATA([monkey2s.conf],
+[[# second monkey R.M. config
+xa_open/-3
+xa_close/0
+]])
+AT_DATA([monkey1d.conf],
+[[# third monkey R.M. config
+xa_open/0
+xa_close/0
+]])
+AT_DATA([monkey2d.conf],
+[[# fourth monkey R.M. config
+xa_open/0
+xa_close/0
+]])
+AT_CHECK([lixa_test_exec.sh reset recycle case0054 CASE_PROF_0002 2>&1 | tee $TESTS_TMP_FILE1 ; grep 'lixa_monkeyrm_close' $TESTS_TMP_FILE1 | grep 'ret_cod' | wc -l | grep 4], [0], [ignore], [ignore])
+AT_CLEANUP
+
+AT_SETUP([TX/3.2/0.2])
+# As for TX/3.2/0.0 but profile CASE_PROF_0000 is specified by LIXA_PROFILE
+# and overrides the CASE_PROF_0001 specified by the program
+AT_DATA([monkey1s.conf],
+[[# first monkey R.M. config
+xa_open/0
+xa_close/0
+]])
+AT_DATA([monkey2s.conf],
+[[# second monkey R.M. config
+xa_open/-3
+xa_close/0
+]])
+AT_DATA([monkey3s.conf],
+[[# third monkey R.M. config
+xa_open/0
+xa_close/0
+]])
+AT_CHECK([export LIXA_PROFILE=CASE_PROF_0000; lixa_test_exec.sh reset recycle case0054 CASE_PROF_0001 2>&1 | tee $TESTS_TMP_FILE1 ; grep 'lixa_monkeyrm_close' $TESTS_TMP_FILE1 | grep 'ret_cod' | wc -l | grep 3], [0], [ignore], [ignore])
+AT_CLEANUP
+
+AT_SETUP([TX/3.2/0.3])
+# As for TX/3.2/0.0 but profile MISSING_PROF is specified by the program
+AT_CHECK([lixa_test_exec.sh reset recycle case0054 MISSING_PROF 2>&1], [1], [ignore], [ignore])
+AT_CLEANUP
+
+AT_SETUP([TX/3.2/0.4])
+# As for TX/3.2/0.0 but profile CASE_PROF_0000 is specified by LIXA_PROFILE
+# and overrides the MISSING_PROF specified by the program
+AT_DATA([monkey1s.conf],
+[[# first monkey R.M. config
+xa_open/0
+xa_close/0
+]])
+AT_DATA([monkey2s.conf],
+[[# second monkey R.M. config
+xa_open/-3
+xa_close/0
+]])
+AT_DATA([monkey3s.conf],
+[[# third monkey R.M. config
+xa_open/0
+xa_close/0
+]])
+AT_CHECK([export LIXA_PROFILE=CASE_PROF_0000; lixa_test_exec.sh reset recycle case0054 MISSING_PROF 2>&1 | tee $TESTS_TMP_FILE1 ; grep 'lixa_monkeyrm_close' $TESTS_TMP_FILE1 | grep 'ret_cod' | wc -l | grep 3], [0], [ignore], [ignore])
+AT_CLEANUP
+
+AT_SETUP([TX/3.2/0.2])
```

Comment:
This should be TX/3.2/0.5 because TX/3.2/0.2 already exists.

Reply:
Sorry, fixed!